pkg/services/k8s/resources/accesspolicy/accesspolicy.crd.yml

```diff
@@ -0,0 +1,64 @@
+# Code generated - EDITING IS FUTILE. DO NOT EDIT.
+#
+# Generated by:
+#     kinds/gen.go
+# Using jennies:
+#     YamlCRDJenny
+#
+# Run 'make gen-cue' from repository root to regenerate.
+
+kind: CustomResourceDefinition
+apiVersion: apiextensions.k8s.io/v1
+metadata:
+    name: accesspolicys.accesspolicy.core.grafana.com
+spec:
+    group: accesspolicy.core.grafana.com
+    versions:
+        - name: v0-0alpha1
+          served: true
+          storage: true
+          schema:
+            openAPIV3Schema:
+                properties:
+                    spec:
+                        properties:
+                            role:
+                                description: |-
+                                    UID for user/team/service...
+                                    ??? should this be a #ResourceID  but kind=user|team|???
+                                type: string
+                            rules:
+                                items:
+                                    properties:
+                                        action:
+                                            type: string
+                                        what:
+                                            type: string
+                                    required:
+                                        - what
+                                        - action
+                                    type: object
+                                type: array
+                            scope:
+                                description: The entity this policy applies to
+                                properties:
+                                    kind:
+                                        type: string
+                                    name:
+                                        type: string
+                                required:
+                                    - kind
+                                    - name
+                                type: object
+                        required:
+                            - role
+                            - scope
+                            - rules
+                        type: object
+                required:
+                    - spec
+                type: object
+    names:
+        kind: AccessPolicy
+        plural: accesspolicys
```

Comment:
accesspolicies

Reply:
🤣 --  @mildwonkey do you know status/thoughts on where these would be explicitly defined rather than auto-generated?

Reply:
(sorry, I'm just seeing this today) Yes, you can include the `pluralName` in the .cue file - it defaults to `name + s `(obviously) but can be overridden.